### PR TITLE
SISRP-17980 - 5.3 - Dashboard (Faculty) - Faculty Resources

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-17980
move faculty resource links to dashboard

### CODE REVIEW

File: src/assets/templates/dashboard.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <div class="medium-4 columns cc-column-last">
     <div data-ng-include="'widgets/activity.html'"></div>
     <div data-ng-include="'widgets/advisor/resources.html'" data-ng-if="api.user.profile.roles.advisor"></div>
+    <div data-ng-if="isInstructorOrGsi" data-ng-include="'faculty-resources.html'"></div>
```
Comment (johncrossman) [inline src/assets/templates/dashboard.html]: @dfkaye Can we move the `data-ng-if` to the end of div element?

File: src/assets/templates/dashboard.html [hunk 2]
```diff
@@ -14,6 +14,7 @@
   <div class="medium-4 columns cc-column-last">
     <div data-ng-include="'widgets/activity.html'"></div>
     <div data-ng-include="'widgets/advisor/resources.html'" data-ng-if="api.user.profile.roles.advisor"></div>
+    <div data-ng-include="'faculty-resources.html'" data-ng-if="isInstructorOrGsi"></div>
```
Comment (christianvuerings) [inline src/assets/templates/dashboard.html]: @dfkaye so this variable `isInstructorOrGsi` is exposed on the main dashboard?

Comment (paulkerschen) [inline src/assets/templates/dashboard.html]: @christianvuerings @dfkaye I don't believe it is available; looks like it's derived from route slugs on to My Academics detail pages. 

If this card is meant to be surfaced only to faculty, then `api.user.profile.roles.faculty` would be the right check. If it's meant to surface to faculty and GSIs both, `api.user.profile.roles.faculty || api.user.profile.hasInstructorHistory` might be the best check available.

Comment (dfkaye) [inline src/assets/templates/dashboard.html]: @christianvuerings @paulkerschen thank you, gentlemen!  I'll make that change in a little bit...

**Comment by johncrossman** [comment]
Thanks @dfkaye

**Comment by johncrossman** [comment]
@dfkaye This will get a QA PR, too?

**Pull request merged**